chef/cookbooks/crowbar/templates/default/crowbar-migrate-keystone-and-start.sh.erb

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# This makes sure openstack related services are really shut down
+# We intend to do this by shutting down pacemaker resources, but if
+# clone_stateless_services is set to false, some services are not managed by pacemaker.
+
+LOGFILE=/var/log/crowbar/node-upgrade.log
+UPGRADEDIR=/var/lib/crowbar/upgrade
+mkdir -p "`dirname "$LOGFILE"`"
+exec >>"$LOGFILE" 2>&1
+
+log()
+{
+    set +x
+    echo "[$(date --iso-8601=ns)] [$$] $@"
+    set -x
+}
+
+log "Executing $BASH_SOURCE"
+
+set -x
+
+mkdir -p $UPGRADEDIR
+rm -f $UPGRADEDIR/crowbar-migrate-keystone-and-startfailed
```

Comment:
I know that you are actually not creating `failed` file in any case (at least for now), but it should be called `crowbar-migrate-keystone-and-start-failed`